conda-lock.yml

```diff
@@ -0,0 +1,4591 @@
+# This lock file was generated by conda-lock (https://github.com/conda-incubator/conda-lock). DO NOT EDIT!
+#
+# A "lock file" contains a concrete list of package versions (with checksums) to be installed. Unlike
+# e.g. `conda env create`, the resulting environment will not change as new package versions become
+# available, unless you explicitly update the lock file.
+#
+# Install this environment as "YOURENV" with:
+#     conda-lock install -n YOURENV --file conda-lock.yml
+# To update a single package to the latest version compatible with the version constraints in the source:
+#     conda-lock lock --lockfile conda-lock.yml --update PACKAGE
+# To re-solve the entire environment, e.g. after changing a version constraint in the source file:
+#     conda-lock -f environment.yml --lockfile conda-lock.yml
+metadata:
+  channels:
+  - url: conda-forge
+    used_env_vars: []
+  - url: nodefaults
+    used_env_vars: []
+  content_hash:
+    linux-64: 27752ab8a15c7be522d4cabfe3ae7906d7ea7cabae18937ecebf74ea2dad3612
+  platforms:
+  - linux-64
```

Comment:
Debating on whether to have a Linux only lockfile, or do macOS and Windows as well :upside_down_face:

Reply:
On second thought, let's just lock it. It adds like 9000+ lines to the `conda-lock.yml` file but at least macOS and Windows users can reproduce the environment fully too.